src/pkt_proc.c

```diff
@@ -895,16 +967,19 @@ void* process_packet (unsigned char *ctx_ptr,
     bool allocated_packet_header = 0;
     uint16_t ether_type = 0,vlan_ether_type = 0, vlan2_ether_type = 0;
     char ipv4_addr[INET_ADDRSTRLEN];
+    char ipv6_addr[INET6_ADDRSTRLEN];
```

Comment:
Isn't this the define you can use above for IPv6 address Len.

Reply:
no, this is the string representation and it is much larger than 128-bits